package dev.ehyeon.hydrangea.user.exception

import dev.ehyeon.hydrangea.common.exception.BaseException

class UserNotFoundException : BaseException("USER_NOT_FOUND")
